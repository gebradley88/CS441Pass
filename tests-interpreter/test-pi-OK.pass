//.TEST -- JWJ
//.Step 1  -- testing floats (before declarations are implemented)
//.source: test-pi-0.cal
//.expected output: test-pi_expected.out
//.this program is easiest (easier  than test-1m.cal - no loop, no int)
//.REMOVE THE ABOVE LINES IF COMMENTS ARE NOT IMPLEMENTED!!!

//compute pi using fractions
{
	double pi_1;
	double  pi_2;
	double pi;
	pi = 3.14159265359;
	pi_1 = 22.0/7.0;
	print pi_1;
	pi_2 = 355.0/113.0;
	print pi_2;
	print (pi_1 -pi_2);
	print pi;
	print (pi -pi_2);
}

