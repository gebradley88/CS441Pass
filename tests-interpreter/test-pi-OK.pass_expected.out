3.14286
3.14159
0.00126422
3.14159
-2.66764e-07
